### PR TITLE
Allow maintainers to toggle other roles and redirect logged in user to landing if they access login/signup pages

### PR DESCRIPTION
**Pull request by venuslimm**
This PR:
- Allow maintainers to toggle other users' roles
- Remove user MongoDB files
- Dont allow logged in user to access login and signup pages by checking jwt token. Logged in users are redirected to /landing.
  - Cannot use cookie to check because if cookie is used to check and user self-adds a cookie to act like they are logged in, it will create a loop.
    - E.g. when user with cookie (but invalid jwt) attempts to access login/signup page, user will be redirected back to the landing page as the user has a cookie but then redirect back to login page since user does not have valid jwt and then redirect to the landing page...
  - However it renders the login/signup page before redirecting user to /landing
    - Solved by checking the absence of cookie before rendering login/signup
    - Not sure if this is the best method because login/signup page will be empty if user has cookie but invalid jwt. But I don't think it will make sense to check the validity of jwt 
- Update `README.md`

### CODE REVIEW

**Review by JrmCkh — APPROVED** [review]
LGTM

File: backend/server/repositories/userRepoMySql.js [hunk 1]
```diff
@@ -191,6 +191,32 @@ const getUsernameFromEmail = (email) => {
...
+    .query(_sql, _placeholders)
+    .then(([result, fields]) => {
+      _success = result.affectedRows === 1;
+    })
+    .catch(console.error);
+
+  await query; // Wait for user to be updated
+  return _success;
+};
+
```
Comment (lhy-hoyin) [inline backend/server/repositories/userRepoMySql.js:219]: Bug might possibly originate from here.

Are you updating the local (i.e cookie's) userId after toggling?

**Review by lhy-hoyin — CHANGES_REQUESTED** [review]
UI look good to me.

But there's bugs relating to the toggling of user role, as well as other bugs here and there.

**Comment by lhy-hoyin** [comment]
Other bugs that need resolving:
* npm start in Auth is still using `nodemon AuthService.js `. Assist to update to `node AuthService.js`
* Check existence of parameter for /auth/generate (see #174)
* Bug from previous push: #175

File: backend/server/repositories/userRepoMySql.js (hunk 1 above)
Comment (lhy-hoyin) [reply backend/server/repositories/userRepoMySql.js:219]: Possibly linked to #173, #176

Comment (venuslimm) [reply backend/server/repositories/userRepoMySql.js]: Thank you, resolved!

**Comment by venuslimm** [comment]
Resolved #173 , #175 , #176

**Comment by venuslimm** [comment]
I have also updated the file structure to follow ass-4

File: backend/server/repositories/userRepoMySql.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const findByEmail = async (email) => {
     .catch(console.error);
 
   await query; // Wait for uid and isMaintainer to be updated
-  return { userId: _userId, isMaintainer: _isMaintainer };
+  return { _userId: _userId, _isMaintainer: _isMaintainer };
```
Comment (lhy-hoyin) [inline backend/server/repositories/userRepoMySql.js:31]: At least for myself, I tend to use leading/trailing underscore for variable names for when the variable is (1) same name as method parameter (2) the variable is used internally only (i.e. not referenced outside of the method)

**Review by lhy-hoyin — APPROVED** [review]
LGTM

**Review by cadencjk — APPROVED** [review]
LGTM

**Review by anthonynmh — APPROVED** [review]
LGTM

**Pull request merged**